### ISSUE TITLE
Area control affecting permissions between attackers and defenders

### ISSUE THREAD

**Issue by mayynophobia** [StoneworksMC/War#7]
- **Please describe what you want in this suggestion.**
When the attackers are siezing control of a specific area of a claim, they should be able to do the same things defenders can (like trapping)

- **How should this feature interact with players?**
The abilities of players to interact with the area around them should change depending on area control progress/leading force

<details>
<summary>Further details</summary>

Depending on how the control progress feature is going to work in areas, I think it would be beneficial for permissions in the area to be changed depending on which side is controlling which part of the town (for example, if there's more progress on capturing an area or more players in an area then those players should be given permissions to for example break/place blocks, interact with containers for looting, possibly even ender pearling if the plugin plans related to pearls allow for it). This will hopefully encourage more objective play from pvpers who go to wars purely for gear, and will diversify gameplay throughout a battle

</details>

**Comment by Matt-MX** [comment]
I agree, I think it'd be more fair if either side had the same control over land. It could level the playing field and will change the meta of single sided gank pits for once.

Any damage caused in the war by either side could be reverted back.
